jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -37,7 +37,7 @@
 class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
                          DatasetSelectMixin, AddResultsMixin):
     """
-    See the :ref:`Spectral Extraction Plugin Documentation <spex>` for more details.
+    See the :ref:`Spectral Extraction Plugin Documentation <spectral-extraction>` for more details.
```

Comment:
will changing this break any old links?  I guess as long as this is first introduced in a minor release, then at least links from the app to docs will be ok?

Reply:
The "old link" was broken to begin with, this is a correction.